<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Noto Sans Hatran</family>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Description</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="2,4">
            <item>
             <widget class="QTextBrowser" name="textBrowser">
              <property name="font">
               <font>
                <family>Noto Sans Mahajani</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans Mahajani'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-size:18pt; font-weight:600; color:#e16b8c;&quot;&gt;Instructions&lt;/span&gt;&lt;/p&gt;
&lt;ol style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; font-family:'.AppleSystemUIFont';&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;點選 &amp;quot;Start!&amp;quot; 開始使用&lt;/li&gt;
&lt;li style=&quot; font-family:'.AppleSystemUIFont';&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;先選擇裁判字號&lt;/li&gt;
&lt;li style=&quot; font-family:'.AppleSystemUIFont';&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;點選 “Visualize” 或點擊兩下表格中的內容，跳出新視窗顯示知識圖譜&lt;/li&gt;
&lt;li style=&quot; font-family:'.AppleSystemUIFont';&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;新視窗中的 &amp;quot;Judicial Web&amp;quot; 分頁則可在搜尋框中複製裁判字號以搜尋該判決的詳細內文&lt;/li&gt;
&lt;li style=&quot; font-family:'.AppleSystemUIFont';&quot; style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;點選 “Statistics” 分頁可以年份查詢，並呈現出該年份判決書之統計圖&lt;/li&gt;&lt;/ol&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="textBrowser_2">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="html">
               <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans Hatran'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:14px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-size:18pt; font-weight:600; color:#e16b8c;&quot;&gt;Introduction of Knowledge Graph&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont';&quot;&gt;知識圖譜（Knowledge Graph, KG）是一種表示和組織知識的圖形化結構，由三元組 (Triples) 組成。它使用實體（節點）和實體之間的關係（邊）來表示知識的概念和關聯。知識圖譜是一種圖形數據庫，用於捕捉現實世界中的實體和它們之間的相互作用。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont';&quot;&gt;以下是一個關於洗錢犯罪的三元組例子：&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont';&quot;&gt;主體（Head Entity）：個人（Person）&lt;br /&gt;謂詞（Relation）：犯（suspectedOf）&lt;br /&gt;客體（Tail Entity）：洗錢罪（MoneyLaundering）&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont';&quot;&gt;這個三元組表示了一個主體個人涉嫌洗錢的情況，透過這個關係，我們可以得知這個個人被指控進行洗錢活動。&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont';&quot;&gt;在實際的知識圖譜中，還可以添加更多的相關信息，如洗錢方法、涉案金額、相關法律條文等，以更全面地描述洗錢犯罪相關的知識。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="GoSearch_btn">
            <property name="text">
             <string>Start!</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Search Page</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="3,1">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,20">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>裁判字號：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="JID_combo">
                <property name="currentText">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
              <item>
               <widget class="QPushButton" name="search_btn">
                <property name="text">
                 <string>Visualize</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="exit_btn">
                <property name="text">
                 <string>Exit</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="3,3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Count of Triple:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_total">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,10">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Page:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_page"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="tableView"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPushButton" name="pushButton_first">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normalon>icon/first.png</normalon>
               </iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_previous">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normalon>icon/previous.png</normalon>
               </iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_next">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normalon>icon/next.png</normalon>
               </iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_last">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normalon>icon/last.png</normalon>
               </iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Statistics</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,20,0">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>GroupBox</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_9">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Please select one Year to show the count of judgements by city/county.</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="5,1">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,20">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="text">
                      <string>Year：</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="year_combo">
                     <property name="toolTipDuration">
                      <number>30</number>
                     </property>
                     <property name="currentText">
                      <string>107</string>
                     </property>
                     <item>
                      <property name="text">
                       <string>107</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>108</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>109</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>110</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0">
                   <item>
                    <widget class="QPushButton" name="exit_btn_2">
                     <property name="styleSheet">
                      <string notr="true"/>
                     </property>
                     <property name="text">
                      <string>Exit</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2"/>
          </item>
          <item>
           <widget class="QPushButton" name="news_btn">
            <property name="text">
             <string>查看各法院新聞請按此</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
